src/Http/Controllers/MediaController.php

```diff
@@ -76,17 +77,19 @@ public function deleteCollection(Request $request, MediaService $mediaService):
         return ResponseHelper::createResponseFromArrayResponse($response);
     }
 
-    public function downloadMultiple(Request $request)
+    public function downloadMultiple(DownloadMultipleMediaRequest $request): MediaStream
     {
-        $ids = $request->input('ids');
-        $fileName = Str::finish($request->input('filename') ?? 'media', '.zip');
-        $ids = is_array($ids) ? $ids : explode(',', $ids);
+        $data = $request->validated();
 
-        if (! $ids) {
-            abort(422);
-        }
+        $fileName = Str::finish(data_get($data, 'filename') ?: 'media', '.zip');
+        $ids = is_array(data_get($data, 'ids')) ?
```

Comment:
Usually we begin with the operator in the line...

Reply:
won't do. To me this looks cleaner.